algorithms/unit_tests/CMakeLists.txt

```diff
@@ -35,6 +35,7 @@ foreach(Tag Threads;Serial;OpenMP;Cuda;HPX;HIP;SYCL;OpenMPTarget)
       "#include <Test${Tag}_Category.hpp>\n"
       "#include <TestRandomCommon.hpp>\n"
       "#include <TestSortCommon.hpp>\n"
+      "#include <TestNestedSort.hpp>\n"
```

Comment:
Can you open an issue so we can split this and not end up with a single huge object file.